pkg/controllers/user/networkpolicy/netpol.go

```diff
@@ -292,6 +292,17 @@ func (npmgr *netpolMgr) handleHostNetwork() error {
 	}
 
 	systemNamespaces := GetSystemNamespaces()
+	var systemProjectID string
+	for _, aNS := range namespaces {
+		if systemNamespaces[aNS.Name] {
+			if _, ok := aNS.Labels[nslabels.ProjectIDFieldLabel]; ok {
+				systemProjectID = aNS.Labels[nslabels.ProjectIDFieldLabel]
+			}
+		}
+	}
+	if systemProjectID != "" {
+		npmgr.addUserSystemNamespaces(systemProjectID, systemNamespaces)
```

Comment:
lets incorporate this call to GetSystemNamespaces call. Otherwise it would be easy to mix up

Reply:
I've added `getSystemNamespaces` to this file since it's only used by `netpol` and to add user's system Namespaces - I need access to `npmgr` for listing all ns. 

Letting the generic `GetSystemNamespaces` stay in utils for now.